docs/pages/setup/guides/terraform-provider.mdx

```diff
@@ -0,0 +1,236 @@
+---
+title: Terraform Provider
```

Comment:
Since the guide seems to be OpenSSH specific, maybe reflect that in the title? Something like `Adding OpenSSH nodes with Terraform Provider`.

Reply:
it's not open SSH specific actually :) why did you get that impression?

Reply:
@klizhentas Just reading right below:

```
This section will cover:
- Configuring OpenSSH server `sshd` to join a Teleport cluster...
- Configuring OpenSSH client `ssh` to login into nodes inside a Teleport  cluster.
```

Reply:
Fixed